### PR TITLE
edgesite: optimize code for edgeSiteCommand

### PR DESCRIPTION
**Pull request by TianTianBigWang**
Signed-off-by: TianTianBigWang <zilong.wang@daocloud.io>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: edgesite/cmd/edgesite/app/server.go [hunk 1]
```diff
@@ -61,8 +61,7 @@ runs on edge nodes and manages containerized applications.`,
 			gis.Start(beehiveContext.Done())
 			registerModules(config)
 			// start all modules
-			core.StartModules()
-			core.GracefulShutdown()
+			core.Run()
```
Comment (Iceber) [inline edgesite/cmd/edgesite/app/server.go:64]: It might be clearer to split it into two calls? I'm not sure.

Comment (zc2638) [reply edgesite/cmd/edgesite/app/server.go:64]: The `Run` method combines two methods and has full comment

Comment (fisherxu) [reply edgesite/cmd/edgesite/app/server.go:64]: Looks good.

**Pull request merged**